docs/content/dagster-cloud/deployment/hybrid.mdx

```diff
@@ -1,7 +1,74 @@
 ---
-title: "Hybrid deployment in Dagster Cloud | Dagster Docs"
-
+title: Hybrid deployment in Dagster Cloud | Dagster Docs
 noindex: True
 ---
 
 # Hybrid deployment in Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+In this guide, we'll cover how Hybrid deployment works in Dagster Cloud, including its system architecture, agent, and security.
+
+---
+
+## Hybrid architecture overview
+
+A **hybrid deployment** utilizes a combination of your infrastructure and Dagster-hosted backend services.
+
+The Dagster backend services - including the web frontend, GraphQL API, metadata database, and daemons (responsible for executing schedules and sensors) - are hosted in Dagster Cloud. You are responsible for running an [agent](/dagster-cloud/deployment/agents) in your environment.
+
+<Image
+alt="Dagster Cloud Architecture Diagram"
+src="/images/dagster-cloud/deployment/hybrid-architecture.png"
+width={1583}
+height={765}
+/>
+
+When users interact with the web frontend, queries are made against the GraphQL API, or schedules and sensors tick. This enqueues work for your agent. The agent polls the agent API to see if any work needs to be done and launches user code as appropriate to fulfill requests. User code then streams metadata back to the agent API (GraphQL over HTTPS) to make it available in Dagster Cloud.
```

Comment:
This sentence got broken up

> When users interact with the web frontend, queries are made against the GraphQL API, or schedules and sensors tick, work is enqueued for your agent.

It's not a great one, maybe there's a different way to phrase it

Reply:
Good catch - I was pretty fried when I did this last night. Maybe something like:

```
Work is enqueued for your agent when:

- Users interact with the web front end,
- The GraphQL API is queried, or
- Schedules and sensors tick
```